### PR TITLE
Remove debug statement.

### PR DESCRIPTION
**Pull request by ekoval**
#16

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # [Changelog](https://github.com/yola/healthcheck)
 
+## 0.0.6
+* Remove debug statement.
```
Comment (adrianmoisey) [inline CHANGELOG.md]: What about:
> Bump version in order to remove mistakenly added debug code that was never committed to git.

File: healthcheck/contrib/django/status_endpoint/views.py [hunk 2]
```diff
@@ -22,7 +22,6 @@ class JsonResponseServerError(JsonResponse):
 @require_http_methods(['GET'])
 def status(request):
     checks = []
-
```
Comment (adrianmoisey) [inline healthcheck/contrib/django/status_endpoint/views.py]: Keep this space?

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by jouberthenk** [comment]
:+1:

**Pull request merged**